COPYRIGHT

```diff
@@ -0,0 +1,15 @@
+
+Most of the files are
+
+  Copyright (c) 2004-2015 Odoo S.A.
```

Comment:
2015 :D ?

Reply:
I just copy / paste from the odoo repo, not sure if I can change it or not :/

Reply:
I suppose we can say "TODAY" without that much impact.

Reply:
Indeed, looks like the date is the "claim date", (the plugin didn't exists in 2014) https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

So I update it :)

Reply:
Shouldn't it start on 2020 or 2021 ?

Reply:
IDK if we can put a year in the past...

Reply:
ok, thanks for the explanation :)